tests/consensus_tests/utils.py

```diff
@@ -19,7 +19,8 @@
 def every_test():
     yield
     print()
-    for p in processes:
```

Comment:
Is this the issue the reason we had flaky tests lately on CI?

Reply:
No, this is just a print fix. The processes were all killed after each test also previously, but their ids were still printed in logs

Reply:
This PR should have removed most of the problems with python tests - https://github.com/qdrant/qdrant/pull/782